teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Teacher < Person
+  attr_reader :specialization
+
+  def initialize(specialization:, name: 'Unknown', age: nil, parent_permission: true)
+    super(name: name, age: age, parent_permission: parent_permission)
+    @specialization = specialization
+  end
+
+  def can_use_services?
+    true
+  end
+end
```

Comment:
### README FILE

- Please, let's kindly update our readme by doing the following to make it more professional:
   - Add the project name 
   - Could you add some future features about the project instead of the current placeholder you've added?
   - Add some FAQs and answers or remove the section from the readme since it is optional.
   - Update the client and server sections in the tech stack.
   - Remove the Live Demo since you won't deploy the app.
   - Update the Getting Started section by adding the corresponding instructions on how users can have your app run locally on their machine, etc. how to clone the app, etc. 
   - Add some key features about the app.